enhancements/machine-api/azure-ultra-disks.md

```diff
@@ -132,8 +136,26 @@ type DataDisk struct {
 	// +optional
 	// +kubebuilder:validation:Enum=None;ReadOnly;ReadWrite
 	CachingType CachingTypeOption `json:"cachingType,omitempty"`
+	// DeletionPolicy specifies the data disk deletion policy upon Machine deletion.
+	// Possible values include:
+	// "Delete": If this value is used, the data disk is deleted when the Machine is deleted.
+	// "Detach": If this value is used, the data disk is detached from the Machine and retained when the Machine is deleted.
```

Comment:
Just as a note, the formatting you have here won't be retained when it comes to the generated API reference, better to avoid colons and use full sentences/paragraphs